dashboard/settings.py

```diff
@@ -130,6 +130,7 @@
                 'dashboard.context_processors.current_user_courses_info',
                 'dashboard.context_processors.last_updated',
                 'dashboard.context_processors.get_build_info',
+                'dashboard.context_processors.get_user_defaults',
```

Comment:
Yeah, in the normal process of browsing it will only do it once. I felt this was better than constantly going out to get it. As far as database calls are concerned I think we'll have to figure out the best way to cache those but that's a different problem.